### PR TITLE
yabai: 3.2.0 -> 3.2.1

### PR DESCRIPTION
**Pull request by cmacrae**
###### Motivation for this change
New release :)

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/os-specific/darwin/yabai/default.nix [hunk 1]
```diff
@@ -26,7 +26,7 @@ stdenv.mkDerivation rec {
     '';
     homepage = "https://github.com/koekeishiya/yabai";
```
Comment (teto) [inline pkgs/os-specific/darwin/yabai/default.nix:27]: could you add the description as well

Comment (cmacrae) [reply pkgs/os-specific/darwin/yabai/default.nix:27]: There's already description

Comment (teto) [reply pkgs/os-specific/darwin/yabai/default.nix:27]: damn sry.

Comment (cmacrae) [reply pkgs/os-specific/darwin/yabai/default.nix:27]: All good 👍 Easy to miss when GitHub collapses non-differing portions of the file :)

**Comment by cmacrae** [comment]
Any chance this could get a review? 🙏

**Comment by cmacrae** [comment]
Thanks @teto! 🤗

**Pull request merged**